electron_app/AuthLoginModule.js

```diff
@@ -0,0 +1,99 @@
+const electronOauth2 = require('electron-oauth2');
+const constants = require('./helpers/enviroment').constants;
+
+class LoginGoogle {
```

Comment:
Maybe `GoogleLoginWindow` would be a better name for this? The filename also doesn't seem to match the current naming convention for these files.